src/debug/HyprCtl.cpp

```diff
@@ -1677,6 +1677,8 @@ std::string CHyprCtl::getReply(std::string request) {
 
     std::string result = "";
 
+    request = request.substr(0, request.find(" "));
```

Comment:
Is that intended?

Reply:
It's there so it doesn't throw invalid request if we have an additional arg

Reply:
oops, that actually breaks all other commands. fixed